modules/cognito/variables.tf

```
@@ -51,3 +51,9 @@ variable "saml_metadata_source" {
   type        = string
 }
 
+variable "token_validity_duration" {
```

Because this applies only to the web app client I think we should name it specifically

```suggestion
variable "web_access_token_validity_duration" {
```

---

Can you also expose this configuration through to the main module vars so that it can be adjusted in a simple deployment